### PR TITLE
TMGR-8: Add button with actions

### PR DESCRIPTION
**Pull request by acrossoffwest**

### CODE REVIEW

File: src/components/UIElements/ButtonWithActions.vue [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			}
+		},
+		data () {
+			return {
+				showDropdownButtons: false,
+				dropdownOffset: 0,
+				classesForDirection: {
+					down: `absolute z-50 w-full bg-${this.color}-700`,
+					up: `absolute z-50 w-full bg-${this.color}-700`
+				}
```
Comment (savayer) [inline src/components/UIElements/ButtonWithActions.vue:83]: don't pass such props that will change TW classes. Just pass TW class instead

File: src/components/UIElements/ButtonWithActions.vue [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	}
+	.hover\:rounded-l-lg {
+		border-top-left-radius: 0.5rem; /* 8px */
+		border-bottom-left-radius: 0.5rem; /* 8px */
+	}
+	.hover\:rounded-r-lg {
+		border-top-right-radius: 0.5rem; /* 8px */
+		border-bottom-right-radius: 0.5rem; /* 8px */
+	}
+</style>
```
Comment (savayer) [inline src/components/UIElements/ButtonWithActions.vue:120]: such a mess

**Review by savayer — CHANGES_REQUESTED** [review]
There are things to fix obviously, but the main thing is i don't think we need this button, at least right now. Let's put it on hold

File: src/components/UIElements/ButtonWithActions.vue (hunk 1 above)
Comment (acrossoffwest) [reply src/components/UIElements/ButtonWithActions.vue]: I fixed that

File: src/components/UIElements/ButtonWithActions.vue (hunk 2 above)
Comment (acrossoffwest) [reply src/components/UIElements/ButtonWithActions.vue]: removed

**Comment by acrossoffwest** [comment]
I decided to leave the component as it is and in further, I'll figure out how to use it.

**Pull request merged**